docs/en/peripherals/dshot.md

```diff
@@ -147,3 +147,21 @@ ERROR [dshot] No data received. If telemetry is setup correctly, try again.
 
 Check manufacturer documentation for confirmation/details.
 :::
+
+## Bidirectional DShot (Telemetry)
```

Comment:
FYI, added "Telemetry" because when I see bidirectional applied to an ESC, I see "a motor where you can switch the direction", and that is always true in dshot.